### PR TITLE
(GH-128) replaced link to Gitter

### LINKED ISSUE #128: Replace Gitter with GitHub Discussions
The Discussion platform for Cake-Contrib has been moved from Gitter to GitHub Discussion (see cake-build/website#1470). 
Please check this repo for references to Gitte and replace them accordingly.

Search suggests that references could be in

 - README.md
 - docs/input/_Bottom.cshtml

### PR DESCRIPTION
**Pull request by nils-a**
with a link to GitHub discussions

fixes #128

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,11 +26,11 @@ More about Cake at [cakebuild.net](http://cakebuild.net).
...
 - [Documentation](https://cake-contrib.github.io/Cake.Npm)
 
-## Chat Room
+## Discussion
 
-Come join in the conversation about Cake.Yeoman in our Gitter Chat Room.
+If you have questions, search for an existing one, or create a new discussion on the Cake GitHub repository, using the `extension-q-a` category.
 
-[![Join the chat at https://gitter.im/cake-contrib/Lobby](https://badges.gitter.im/cake-contrib/Lobby.svg)](https://gitter.im/cake-contrib/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join in the discussion on the Cake repository](https://img.shields.io/badge/GitHub-Discussions-green?logo=github)](https://github.com/cake-build/cake/discussions)
```
Comment (pascalberger) [inline README.md:33]: Link should go to the correct category: https://github.com/cake-build/cake/discussions/categories/extension-q-a

File: README.md [hunk 2]
```diff
@@ -26,11 +26,11 @@ More about Cake at [cakebuild.net](http://cakebuild.net).
 
 - [Documentation](https://cake-contrib.github.io/Cake.Npm)
 
-## Chat Room
+## Discussion
 
-Come join in the conversation about Cake.Yeoman in our Gitter Chat Room.
+If you have questions, search for an existing one, or create a new discussion on the Cake GitHub repository, using the `extension-q-a` category.
```
Comment (pascalberger) [inline README.md:31]: Use the name of the category shown to the user not the URL slug

```suggestion
If you have questions, search for an existing one, or create a new discussion on the Cake GitHub repository, using the `Extension Q&A` category.
```

File: docs/input/_Bottom.cshtml [hunk 3]
```diff
@@ -1,12 +1,10 @@
 <div class="github-button">
     <a href="https://github.com/cake-contrib/Cake.Npm" target="_blank"><i class="fa fa-github"></i> GitHub</a>
 </div>
-<script>
-    ((window.gitter = {}).chat = {}).options = {
-        room: 'cake-contrib/Lobby'
-    };
-</script>
-<script src="https://sidecar.gitter.im/dist/sidecar.v1.js" async defer></script>
+<a class="discussion-button" href="https://github.com/cake-build/cake/discussions" target="_blank">
```
Comment (pascalberger) [inline docs/input/_Bottom.cshtml:4]: Link should go to the correct category: https://github.com/cake-build/cake/discussions/categories/extension-q-a

**Review by pascalberger — CHANGES_REQUESTED** [review]

**Comment by nils-a** [comment]
@pascalberger I have implemented the comments.

**Review by pascalberger — APPROVED** [review]
LGTM

**Comment by pascalberger** [comment]
@nils-a your changes have been merged, thanks for your contribution 👍

**Pull request merged**